From: Rene van Paassen <Rene.vanPaassen@gmail.com>
Date: 2014-03-18T11:11:23+0100
Subject: gcc on SLES needs explicit template in Irrlicht renderer
References: none
Upstream: no, will go away with SLES

gcc needs template parameter specified.

Index: cegui-0.8.3/cegui/src/RendererModules/Irrlicht/TextureTarget.cpp
===================================================================
--- cegui-0.8.3.orig/cegui/src/RendererModules/Irrlicht/TextureTarget.cpp
+++ cegui-0.8.3/cegui/src/RendererModules/Irrlicht/TextureTarget.cpp
@@ -63,13 +63,13 @@ IrrlichtTextureTarget::~IrrlichtTextureT
 void IrrlichtTextureTarget::activate()
 {
     d_driver.setRenderTarget(d_texture, false, false);
-    IrrlichtRenderTarget::activate();
+    IrrlichtRenderTarget<TextureTarget>::activate();
 }
 
 //----------------------------------------------------------------------------//
 void IrrlichtTextureTarget::deactivate()
 {
-    IrrlichtRenderTarget::deactivate();
+    IrrlichtRenderTarget<TextureTarget>::deactivate();
     d_driver.setRenderTarget(0, false, false);
 }
 
